docs/shared/getting-started/intro.md

```diff
@@ -1,118 +1,85 @@
 ---
 title: Intro to Nx
-description: Nx is a powerful open-source build system that enhances developer productivity, optimizes CI performance, and maintains code quality with features like task caching and distribution.
+description: Nx is a powerful build system that streamlines development workflows, reduces errors, and dramatically cuts waste in projects of any scale.
```

Comment:
should this include our newer AI based tag lines in anyway ?

> An AI-first build platform that connects everything from your editor to CI.
Helping you deliver fast, without breaking things.